src/concurrently.ts

```diff
@@ -30,7 +30,7 @@ const defaults: ConcurrentlyOptions = {
  * If value is a string, then that's the command's command line.
  * Fine grained options can be defined by using the object format.
  */
-export type ConcurrentlyCommandInput = string | Partial<CommandInfo>;
+export type ConcurrentlyCommandInput = string | ({ command: string } & Partial<CommandInfo>);
```

Comment:
bit weird to allow a command not to have a command...

Reply:
Nice 😄